helm/cert-exporter/templates/np-deployment.yaml

```diff
@@ -21,6 +21,9 @@ spec:
     - podSelector:
         matchLabels:
           app: prometheus
+    - podSelector:
+        matchLabels:
+          app.kubernetes.io/name: api-server
```

Comment:
I doubt this will work, expecially with cilium.
API server runs on the host network and this makes pod selector rules not to work. Was this tested?

Reply:
we did tested it, yesterday. See related issue. It works for the MC.

Reply:
> Can you test in on a v19-beta1 aws workload cluster? It shouldn't work

Sure I can, but this fix is only meant to solve a problem in the MCs. As far as I saw, WCs are not affected. The fix itself is not a problem for WCs as there is a cilium policy also in place.

Reply:
> meaning all ingress is allowed from the nodes.

now that I read this again, something changed then, because the issue is that the api-server was not able to reach the cert-exporter and net-exporter pods. After updating the policy, it started to work.
I'll dig a bit more then.

Reply:
in beta1 the deftault policies were removed, each app has to define their full need in their own network policies

Reply:
I mean that something changed in the MCs. What changed in beta1 is not really relevant (customer has no cluster in v19-beta1)

Reply:
I tested it today with v19. The PR has no impact in WCs, so we should be good to proceed.